src/ADSL.xml

```diff
@@ -0,0 +1,14 @@
+<SPDX name="Amazon Digital Services License" identifier="ADSL" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/AmazonDigitalServicesLicense</url>
+  </urls>
+  <license>
+    <body>
+      <p>This software code is made available "AS IS" without warranties of any kind. You may copy, display,
+         modify and redistribute the software code either by itself or as incorporated into your code; provided
+         that &gt; you do not remove any proprietary notices. Your use of this software code is at your own
+         risk and you waive any claim against Amazon Digital Services, Inc. or its affiliates with respect to
+         your use of this software code. (c) 2006 Amazon Digital Services, Inc. or its affiliates.</p>
```

Comment:
Is it permitted to use a <copyright> tag within <body>, so as to capture the copyright here?